torch/_inductor/decomposition.py

```diff
@@ -559,6 +559,11 @@ def bernoulli_(self, p=0.5):
     return self.copy_(torch.rand_like(self, dtype=torch.float32) < p)
 
 
+@register_extra_random_decomp([aten.uniform_])
+def uniform_(self, low=0, high=1):
+    return self.copy_((high - low) * torch.rand_like(self) + low)
```

Comment:
This formula may overflow. Please consider using a formula that does not overflow regardless of the values of `high` and `low`.

Reply:
Have a link?

Reply:
xref. https://github.com/pytorch/pytorch/issues/90815#issuecomment-1354403724

Reply:
Hi all, I believe we have aligned on this, https://github.com/pytorch/pytorch/issues/90815#issuecomment-1356854711.